mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/com/liferay/todo/TodoPortlet.java

```diff
@@ -92,6 +93,37 @@ public void doView(
 			super.doView(renderRequest, renderResponse);
 	}
 
+	@Override
+	public void serveResource(
+			ResourceRequest resourceRequest, ResourceResponse resourceResponse)
+		throws IOException, PortletException {
+			try {
+				if (resourceRequest.getResourceID().equals("task-resource")) {
```

Comment:
@mateomustapic keep in mind that we try to avoid chaining. If you ever doubt chaining or not - choose NOT CHAINING